ProteoformSuiteInternal/TopDownProteoform.cs

```diff
@@ -0,0 +1,111 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+using Proteomics;
+
+namespace ProteoformSuiteInternal
+{
+    public class TopDownProteoform : Proteoform
```

Comment:
I'm curious whether this should inherit ExperimentalProteoform. If not, maybe we should rename ExperimentalProteoform as ExperimentalIntactProteoform and this as ExperimentalTopDownProteoform or something.